assets/css/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: Poppins;
+  src: url(../fonts/Poppins/Poppins-SemiBold.ttf);
+}
+
+@font-face {
+  font-family: PoppinsLight;
+  src: url(../fonts/Poppins/Poppins-Light.ttf);
+}
+
+body {
+  font-family: 'Poppins', Tahoma, Geneva, Verdana, sans-serif;
+  background: #e5e5e5;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 15px;
+  height: 90vh;
+  border-bottom-left-radius: 50px;
+  background: white;
```

Comment:
- [ ] I think there is an issue with the background of the section because it's only until the middle and it should take go after the social icons, like the Figma design :smiley: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/138344790-859760d3-cfcf-4b3d-8a0f-5637a4a2aef3.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/138344825-355cdb18-9674-40d2-ae69-07b81a65fd7f.png)